glslc/test/working_directory.py

```diff
@@ -207,11 +157,63 @@ class TestWorkDirCompileFileAbsolutePath(expect.ValidObjectFile):
     glslc_args = ['-c', '-working-directory=subdir', shader]
 
 
-@inside_glslc_testsuite('WorkDir')
-class TestWorkDirLinkFileAbsolutePath(expect.ValidNamedObjectFile):
-    """Tests -working-directory=<dir> when linking input file with absolute
-    path."""
+# The -working-directory flag should not affect the placement of the link file.
+# The following tests ensure that.
 
-    shader = FileShader('void main() {}', '.vert')
-    glslc_args = ['-working-directory=subdir', shader]
+class WorkDirDoesntAffectLinkedFile(expect.ValidNamedObjectFile):
+    """A base class for tests asserting that -workind-directory has no impact
```

Comment:
I tried it, and it doesn't:
```
deki@deki1>ls subdir/
x.c
deki@deki1>ls a.out 
ls: cannot access a.out: No such file or directory
deki@deki1>clang-3.5 -working-directory subdir x.c 
deki@deki1>ls subdir/
x.c
deki@deki1>ls a.out 
a.out*
deki@deki1>
```

See also https://github.com/google/shaderc/pull/5/files#diff-1f63fb1c30dcf688e73a16603d4169daR111